src/commands/cmd_memory.c

```
@@ -11,6 +11,307 @@
+	int64_t samples          // #samples per relationship type to collect
+) {
+	// compute average node attribute memory consumption
+	int     n_lbls            = Graph_LabelTypeCount(g);
+	int64_t n_nodes           = Graph_NodeCount(g);     // number of nodes
+	int64_t sample_size       = MIN(n_nodes, samples);  // sample size
+	int64_t nodes_sample_size = sample_size;            // nodes sample size
+	int64_t n_labeled_nodes   = 0;                      // #labeled nodes
+	size_t  node_memory_usage = 0;                      // node memory usage
+
+	// determine if graph contains any overlaping nodes
+	// e.g. CREATE (n:A:B)
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)